packages/web-twig/src/Resources/components/Link/stories/LinkUnderline.twig

```diff
@@ -0,0 +1,21 @@
+<h3 style="margin: 0">Always</h3>
```

Comment:
why do we need this?

And if we really do, why not use utility class?

Reply:
Yes, there was weird spacing without this, so I changed it to utility classes